bin/run-prod.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash -ex
 
 ./bin/run-common.sh
-exec gunicorn standup.wsgi:application -b 0.0.0.0:${PORT:-8000} --log-file -
+exec newrelic-admin run-program gunicorn standup.wsgi:application -b 0.0.0.0:${PORT:-8000} --log-file -
```

Comment:
What does this do if you don't have a valid new-relic account or creds or something? Does it prevent Standup from running?